### ISSUE TITLE
Failed set beneficiary request

### ISSUE THREAD

**Issue by etherunit** [mysteriumnetwork/node#5219]
0x22a150e7983abdd0605bb4279076dce947c12ee1 is trying to set beneficiary. 
OS: Alpine Linux v3.14(docker)
Node v. 1.9
This issue was reported 4 times today after switch to 1.9.

```
2022-06-17T07:43:47.061 WRN ../../session/pingpong/consumer_balance_tracker.go:475 > Could not get consumer channel error="no contract code at given address"

2022-06-17T07:43:49.100 DBG ../../core/location/oracle_resolver.go:65 > Detecting with oracle resolver

2022-06-17T07:43:51.221 DBG ../../eventbus/event_bus.go:101 > Published topic="NAT-type-detected" event=prcone

2022-06-17T07:43:51.221 DBG ../../config/config.go:215 > Returning CLI value launcher.ver:1.0.30/windows

2022-06-17T07:43:52.253 DBG ../../eventbus/event_bus.go:101 > Published topic="location-update-event" event={IP:213.64.96.65 ASN:3301 ISP:Telia Network Services Continent:EU Country:SE City:Stockholms lan IPType:residential}

2022-06-17T07:43:53.923 INF ../../session/pingpong/pricing.go:177 > pricing info loaded. expires @ 2022-06-17 07:46:17.225447889 +0000 UTC

17T07:45:13.095 ERR ../../session/pingpong/hermes_promise_settler.go:1008 > Could not update promise fee error="could not fetch settle fees: server responded with an error: 500 (https://transactor.mysterium.network/api/v1/fee/0/settle) [err_transactor_fetch_fees] no fees known for chain 0: redis: nil"

2022-06-17T07:45:13.106 ERR ../../tequilapi/endpoints/transactor.go:880 > Failed set beneficiary request for ID: 0x22a150e7983abdd0605bb4279076dce947c12ee1, &{ProviderID:0x22a150e7983abdd0605bb4279076dce947c12ee1 HermesID:0x80Ed28d84792d8b153bf2F25F0C4B7a1381dE4ab Beneficiary:0x5172Fb092E5EC28e0eD3ce42F312aF199E1dAd4D} error="could not fetch settle fees: server responded with an error: 500 (https://transactor.mysterium.network/api/v1/fee/0/settle) [err_transactor_fetch_fees] no fees known for chain 0: redis: nil"
```

**Comment by tomasmik** [comment]
Working on it.

**Comment by Guillembonet** [comment]
Fixed in: #5220 
Needs further improvements, so I will leave it open. There might still be some edge cases for which it fails.

**Comment by tomasmik** [comment]
Fixed.
